.github/workflows/php-static.yml

```diff
@@ -0,0 +1,96 @@
+name: Static analysis
+
+on:
+ workflow_call:
+  inputs:
+   php-versions:
+    required: true
+    type: string
+
+jobs:
+  phpstan:
+    strategy:
+      matrix:
+        php: ${{ fromJSON(inputs.php-versions) }}
```

Comment:
so what do we want to do here ? @jerotire  thoughts having proposed we have the matrix ? Did you envisage us having some conditional rules ?  I am not sure I really questioned it

Reply:
Yeah, @JoshuaLicense makes sense. No need for different PHP versions for static analysis. I do not envisage us having conditional rules in the future, perhaps a little now since some projects are using different versions of PHP which some later features being unsupported in our current versions. But, that should not be an issue later right? 😄

Reply:
build for the now 👍